### PR TITLE
Plone deployment training minor fix proposal

### PR DESCRIPTION
**Pull request by deruosi**
- some minor fix to ease following the training istructions (IMHO)
- added a troubleshooting paragraph with sone other reported issues

### CODE REVIEW

File: docs/plone-deployment/setup.md [hunk 1]
```diff
@@ -105,3 +111,28 @@ The latest version of {term}`Cookiecutter` will be used, and it doesn’t requir
...
 {term}`Make` is pre-installed on most Linux distributions. For macOS, install Xcode and its command-line tools. Windows users are advised to use [WSL](https://learn.microsoft.com/en-us/windows/wsl/install) that includes `make`.
+
+### 9. **Troubleshooting**
+
+#### Insufficient Docker virtual disk space
+
+Docker requires sufficient virtual disk space to install and build images. A typical error message may be: `Could not create directory.` or anything else that mentions writing to the image.
+To resolve this, use an appropriate docker system prune option, such as:
+
+```
```
Comment (stevepiercy) [inline docs/plone-deployment/setup.md:122]:
````suggestion
```shell
````

File: docs/plone-deployment/setup.md [hunk 2]
```diff
@@ -105,3 +111,28 @@ The latest version of {term}`Cookiecutter` will be used, and it doesn’t requir
...
+docker system prune -a
+```
+
+See also [https://docs.docker.com/engine/reference/commandline/system_prune/](https://docs.docker.com/engine/reference/commandline/system_prune/)
```
Comment (stevepiercy) [inline docs/plone-deployment/setup.md:126]: MyST syntax.

````suggestion
```{seealso}
See also https://docs.docker.com/engine/reference/commandline/system_prune/.
```
````

File: docs/plone-deployment/setup.md [hunk 3]
```diff
@@ -105,3 +111,28 @@ The latest version of {term}`Cookiecutter` will be used, and it doesn’t requir
...
+
+You can also configure settings for Docker Desktop. Under `Prefences > Resources > Advanced`, you can configure appropriate settings for virtual disk limit and memory.
```
Comment (stevepiercy) [inline docs/plone-deployment/setup.md:128]:
```suggestion
You can also configure settings for Docker Desktop.
Under {guilabel}`Prefences > Resources > Advanced`, you can configure appropriate settings for virtual disk limit and memory.
```

File: docs/plone-deployment/setup.md [hunk 4]
```diff
@@ -105,3 +111,28 @@ The latest version of {term}`Cookiecutter` will be used, and it doesn’t requir
...
+
+See also:
```
Comment (stevepiercy) [inline docs/plone-deployment/setup.md:130]:
````suggestion
```{seealso}
See also
- [Change Docker Desktop settings on Mac](https://docs.docker.com/desktop/settings/mac/#advanced)
- [Change Docker Desktop settings on Windows](https://docs.docker.com/desktop/settings/windows/#advanced)
- [Change Docker Desktop settings on Linux](https://docs.docker.com/desktop/settings/linux/#advanced)
```
````

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Mostly some MyST syntax fixes and style guide things. Otherwise this is good.

**Comment by deruosi** [comment]
thanks @stevepiercy , I've committed your changes. Have I to *Close with comment* (which seems the only option I have from github) and make a new pull request?

**Comment by stevepiercy** [comment]
Would you please read and follow [Contributing to Plone](https://6.docs.plone.org/contributing/index.html)? I assumed you had already signed the agreement.

File: docs/plone-deployment/project-edit.md [hunk 5]
```diff
@@ -43,7 +43,12 @@ Execute `make install-frontend` from the repository root to install the new add-
 
 #### Restarting the Project
 
-If the project is running, halt the Frontend with {kbd}`Ctrl-C`. Restart it with `make start-frontend` and navigate to [http://localhost:3000](http://localhost:3000). After authentication, the new block becomes available on the content edit page.
+Start your project with: `make start-backend` and `make start-frontend` on different shells.
+
+If the project was running, halt the Frontend with {kbd}`Ctrl-C`. Restart it with `make start-frontend`
```
Comment (stevepiercy) [inline docs/plone-deployment/project-edit.md:48]: The project was stopped in the previous step.

```suggestion
```

File: docs/plone-deployment/project-edit.md [hunk 6]
```diff
@@ -43,7 +43,12 @@ Execute `make install-frontend` from the repository root to install the new add-
 
 #### Restarting the Project
 
-If the project is running, halt the Frontend with {kbd}`Ctrl-C`. Restart it with `make start-frontend` and navigate to [http://localhost:3000](http://localhost:3000). After authentication, the new block becomes available on the content edit page.
+Start your project with: `make start-backend` and `make start-frontend` on different shells.
```
Comment (stevepiercy) [inline docs/plone-deployment/project-edit.md:46]: Duplicated by below line.

```suggestion
```

File: docs/plone-deployment/setup.md [hunk 7]
```diff
@@ -105,3 +111,36 @@ The latest version of {term}`Cookiecutter` will be used, and it doesn’t requir
...
+You can also configure settings for Docker Desktop.
+Under {guilabel}`Prefences > Resources > Advanced`, you can configure appropriate settings for virtual disk limit and memory.
+
+```{seealso}
+See also
+- [Change Docker Desktop settings on Mac](https://docs.docker.com/desktop/settings/mac/#advanced)
+- [Change Docker Desktop settings on Windows](https://docs.docker.com/desktop/settings/windows/#advanced)
+- [Change Docker Desktop settings on Linux](https://docs.docker.com/desktop/settings/linux/#advanced)
+```
+- [Change Docker Desktop settings on Mac](https://docs.docker.com/desktop/settings/mac/#advanced)
```
Comment (stevepiercy) [inline docs/plone-deployment/setup.md:139]: Duplicated above.

```suggestion
```

File: docs/plone-deployment/setup.md [hunk 8]
```diff
@@ -105,3 +111,36 @@ The latest version of {term}`Cookiecutter` will be used, and it doesn’t requir
...
+- [Change Docker Desktop settings on Windows](https://docs.docker.com/desktop/settings/windows/)
```
Comment (stevepiercy) [inline docs/plone-deployment/setup.md:140]: Duplicated above.

```suggestion
```

File: docs/plone-deployment/setup.md [hunk 9]
```diff
@@ -105,3 +111,36 @@ The latest version of {term}`Cookiecutter` will be used, and it doesn’t requir
...
+- [Change Docker Desktop settings on Linux](https://docs.docker.com/desktop/settings/linux/)
```
Comment (stevepiercy) [inline docs/plone-deployment/setup.md:141]: Duplicated above.

```suggestion
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
There were some duplicated lines that need to be removed.

**Comment by deruosi** [comment]
I signed the agreement yesterday, just before accepting and committing the changes

I realize the process is more complex than I thought: sorry for the inconvenience. By the way: next time I will create a new branch even for the (supposed) minor fix, I promise!

**Review by stevepiercy — CHANGES_REQUESTED** [review]
One more minor change, and I approve, pending review and acceptance of your Plone Contributor Agreement. Thank you for your patience!

**Review by stevepiercy — APPROVED** [review]

**Comment by stevepiercy** [comment]
@deruosi this looks good. Please let me know when your Plone Contributor Agreement is approved, then I can merge this PR. Thank you!

**Comment by deruosi** [comment]
@stevepiercy my Plone developer agreement was processed a few hours ago: I just joined the Plone Contributors team. Thanks!

**Comment by stevepiercy** [comment]
@deruosi thank you!

**Pull request merged**